ThirdParty.json

```diff
@@ -2,96 +2,119 @@
   {
     "name": "Async++",
     "url": "https://github.com/Amanieu/asyncplusplus",
-    "license": "MIT"
+    "version": "756893feb9e69c098225d5a454a668a2c139e4be",
+    "license": ["MIT"]
   },
   {
     "name": "Catch2",
     "url": "https://github.com/catchorg/Catch2",
-    "license": "Boost Software License 1.0"
+    "version": "2.13.9",
+    "license": ["Boost Software License 1.0"],
+    "notes": "Only used in the tests."
   },
   {
     "name": "cmake-modules",
     "url": "https://github.com/bilke/cmake-modules",
-    "license": "Boost Software License 1.0"
+    "version": "d6d1a778e41cb114e5cdf279b8a659fa0ce9a0d4",
+    "license": ["Boost Software License 1.0", "BSD-3-Clause"],
+    "notes": "Only used in the build system."
   },
   {
     "name": "cpp-httplib",
     "url": "https://github.com/yhirose/cpp-httplib",
-    "license": "MIT"
+    "version": "0.10.7",
+    "license": ["MIT"]
   },
   {
     "name": "CSPRNG",
     "url": "https://github.com/Duthomhas/CSPRNG",
-    "license": "Boost Software License 1.0"
+    "version": "8768a94b4b04213c0798b80824a04ae4990e9847",
+    "license": ["Boost Software License 1.0"]
   },
   {
     "name": "Draco",
     "url": "https://github.com/google/draco",
-    "license": "Apache 2.0"
+    "version": "1.5.2",
+    "license": ["Apache 2.0"]
   },
   {
     "name": "earcut",
     "url": "https://github.com/mapbox/earcut.hpp",
-    "license": "ISC License"
+    "version": "2.2.3",
+    "license": ["ISC License"]
   },
   {
     "name": "GLM",
     "url": "https://github.com/g-truc/glm",
-    "license": "MIT"
+    "version": "0.9.9.8",
+    "license": ["MIT"]
   },
   {
     "name": "GSL",
     "url": "https://github.com/microsoft/GSL",
-    "license": "MIT"
+    "version": "4.0.0",
+    "license": ["MIT"]
   },
   {
     "name": "KTX-Software",
     "url": "https://github.com/KhronosGroup/KTX-Software",
-    "license": "Apache 2.0"
+    "version": "94764c270d43d90e4e4cedee6248753296f874b3",
+    "license": ["Apache 2.0"],
+    "notes": "Using Cesium's fork of this library, which has minor differences from the upstream: https://github.com/CesiumGS/KTX-Software/tree/cesium-native"
```

Comment:
Should the URL point to Cesium's fork in this case? Thats what we do for [libcitygml](https://github.com/CesiumGS/libcitygml) for example. Of course with LGPL, it is mandatory to point to any custom modifications, so thats the slight difference.

Reply:
I wasn't sure, but I've now made that change.